src/TraceEvent/ETWTraceEventSource.cs

```diff
@@ -592,7 +592,7 @@ private bool ProcessOneFile()
         {
             int dwErr = TraceEventNativeMethods.ProcessTrace(handles, (uint)handles.Length, (IntPtr)0, (IntPtr)0);
             if (dwErr == 6)
-                throw new ApplicationException("Error opening ETL file.  Most likely caused by opening a Win8 Trace on a Pre Win8 OS.");
+                throw new Exception("Error opening ETL file.  Most likely caused by opening a Win8 Trace on a Pre Win8 OS.");
```

Comment:
Same issue with Application Exception.  We should probably make a helper and fix it one place.